src/Authenticators/SAMLAuthenticator.php

```diff
@@ -54,22 +55,19 @@ public static function get_login_form(Controller $controller)
     }
 
     /**
-     * Sends the authentication process down the SAML rabbit hole. It will trigger
-     * the IdP redirection via the 3rd party implementation, and if successful, the user
-     * will be delivered to the SAMLController::acs.
+     * This method does nothing, as all authentication via SAML is handled via HTTP redirects (similar to OAuth) which
+     * are not supported by the Authenticator system. Therefore, SAML authentication is only triggered when a user hits
+     * the SAMLController acs() endpoint when returning from the identity provider.
      *
      * @param array $data
      * @param HTTPRequest $request
      * @param ValidationResult|null $result
      * @return bool|Member|void
```

Comment:
The doc block could use an update

Reply:
I've updated it slightly now, but it's still accurate - the method should do nothing (and this is a major/breaking change because of that fact among others). Is there anything else you want to see in this docblock?

Reply:
Ah sorry, I meant the return tag - it only returns null